test/framework/cluster.go

```diff
@@ -584,29 +587,49 @@ func (e *ClusterE2ETest) WithClusterConfig(fillers ...api.ClusterConfigFiller) *
 	return e
 }
 
-func (e *ClusterE2ETest) ImportImages(opts ...CommandOpt) {
-	importImagesArgs := []string{"import-images", "-f", e.ClusterConfigLocation}
-	e.RunEKSA(importImagesArgs, opts...)
-}
-
 func (e *ClusterE2ETest) DownloadArtifacts(opts ...CommandOpt) {
 	downloadArtifactsArgs := []string{"download", "artifacts", "-f", e.ClusterConfigLocation}
+	if getBundlesOverride() == "true" {
+		downloadArtifactsArgs = append(downloadArtifactsArgs, "--bundles-override", defaultBundleReleaseManifestFile)
+	}
 	e.RunEKSA(downloadArtifactsArgs, opts...)
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	if _, err := os.Stat(defaultDownloadArtifactsOutputLocation); err != nil {
 		e.T.Fatal(err)
 	} else {
-		e.T.Log("Downloaded artifacts saved at eks-anywhere-downloads.tar.gz")
+		e.T.Logf("Downloaded artifacts tarball saved at %s", defaultDownloadArtifactsOutputLocation)
 	}
 }
 
 // ExtractDownloadedArtifacts extract the downloaded artifacts.
 func (e *ClusterE2ETest) ExtractDownloadedArtifacts(opts ...CommandOpt) {
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	e.T.Log("Extracting downloaded artifacts")
+	e.Run("tar", "-xf", defaultDownloadArtifactsOutputLocation)
+}
+
+func (e *ClusterE2ETest) DownloadImages(opts ...CommandOpt) {
+	downloadImagesArgs := []string{"download", "images", "-o", defaultDownloadImagesOutputLocation}
+	if getBundlesOverride() == "true" {
+		downloadImagesArgs = append(downloadImagesArgs, "--bundles-override", defaultBundleReleaseManifestFile)
+	}
+	e.RunEKSA(downloadImagesArgs, opts...)
+	if _, err := os.Stat(defaultDownloadImagesOutputLocation); err != nil {
 		e.T.Fatal(err)
+	} else {
+		e.T.Logf("Downloaded images archive saved at %s", defaultDownloadImagesOutputLocation)
 	}
+}
+
+func (e *ClusterE2ETest) ImportImages(opts ...CommandOpt) {
+	clusterConfig := e.ClusterConfig.Cluster
+	registyMirrorEndpoint, registryMirrorPort := clusterConfig.Spec.RegistryMirrorConfiguration.Endpoint, clusterConfig.Spec.RegistryMirrorConfiguration.Port
+	registryMirrorHost := net.JoinHostPort(registyMirrorEndpoint, registryMirrorPort)
+	importImagesArgs := []string{"import images", "--input", defaultDownloadImagesOutputLocation, "--bundles", defaultBundleReleaseManifestFile, "--registry", registryMirrorHost, "--insecure"}
```

Comment:
import images might fail in airgapped environment if defaultBundleReleaseManifestFile is used, since it may contain url to remote yaml files

Reply:
yeah I changed this logic to use the one from extracted artifacts download.